workers/data_refinery_workers/processors/smasher.py

```diff
@@ -117,20 +119,40 @@ def _prepare_files(job_context: Dict) -> Dict:
     log_state("end prepare files", job_context["job"], start_prepare_files)
     return job_context
 
+def downlad_computed_file(item):
+    """ this function downloads the latest computed file sent in it's arguments,
+    its used to parallelize the download of all files. """
+    (output_file_path, latest_computed_file) = item
+    try:
+        latest_computed_file.get_synced_file_path(path=output_file_path)
+    except:
+        # Let's not fail if there's an error syncing one of the quant.sf files
+        logger.exception('Failed to sync computed file', computed_file_id=latest_computed_file.pk)
+        pass
 
 def sync_quant_files(output_path, files_sample_tuple, job_context: Dict):
     """ Takes a list of ComputedFiles and copies the ones that are quant files to the provided directory.
         Returns the total number of samples that were included """
     num_samples = 0
-    for (_, sample) in files_sample_tuple:
-        latest_computed_file = sample.get_most_recent_quant_sf_file()
-        # we just want to output the quant.sf files
-        if not latest_computed_file: continue
-        accession_code = sample.accession_code
-        # copy file to the output path
-        output_file_path = output_path + accession_code + "_quant.sf"
-        num_samples += 1
-        latest_computed_file.get_synced_file_path(path=output_file_path)
+    samples = [sample for (_, sample) in files_sample_tuple]
+    page_size = 100
+    # split the samples in groups and download each one individually
+    with Pool(processes=PROCESS_POOL_SIZE) as pool:
```

Comment:
I think the more obvious way to do this is remove the context block and clean up explicitly 

```pool = Pool(processes=PROCESS_POOL_SIZE)```
... use pool ...
then at the end you can
```
pool.close()
pool.join()
```